modules/dbsupport/src/main/java/org/jpos/ee/DB.java

```diff
@@ -327,8 +327,11 @@ public synchronized void close() throws HibernateException
     {
         if (session != null)
         {
-            session.close();
-            session = null;
+            synchronized (DB.class)
```

Comment:
This is to guard against this error I was seeing:
```
org.jpos.gl.AccountLockTest > executionError FAILED
    org.hibernate.HibernateException: Detected concurrent management of connection resources. This might indicate a multi-threaded use of Hibernate in combination with managed resources, which is not supported.
        at org.hibernate.resource.jdbc.internal.LogicalConnectionManagedImpl.releaseConnection(LogicalConnectionManagedImpl.java:221)
        at org.hibernate.resource.jdbc.internal.LogicalConnectionManagedImpl.close(LogicalConnectionManagedImpl.java:258)
        at org.hibernate.engine.jdbc.internal.JdbcCoordinatorImpl.close(JdbcCoordinatorImpl.java:175)
        at org.hibernate.internal.AbstractSharedSessionContract.close(AbstractSharedSessionContract.java:342)
        at org.hibernate.internal.SessionImpl.closeWithoutOpenChecks(SessionImpl.java:385)
        at org.hibernate.internal.SessionImpl.close(SessionImpl.java:361)
        at org.jpos.ee.DB.close(DB.java:330)
        at org.jpos.gl.GLSession.close(GLSession.java:1520)
        at org.jpos.gl.TestBase.tearDownBase(TestBase.java:100)
```